### PR TITLE
[feat]: Add quantity

### PR DESCRIPTION
**Pull request by AndrzejMalirz**
Add toast on success

### CODE REVIEW

File: src/app/actions/equipment/a_quantity_add.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      return true;
+      })
+    }),
+    switchMap(() => ws.destroy()),
+    filter(wizardDestroyed => wizardDestroyed.save),
+    switchMap(wizardDestroyed => {
+      const params = new HttpParams();
+      return apis.getEditObjectByID(CID.EQUIPMENT, params, String(vs.objID)).pipe(
+        switchMap(response => {
+          console.log(response);
```
Comment (kamil-oberaj) [inline src/app/actions/equipment/a_quantity_add.ts:30]: redundant log

File: src/app/actions/equipment/a_quantity_add.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          return apis.saveRecord(
+          CID.EQUIPMENT,
```
Comment (Pablo203) [inline src/app/actions/equipment/a_quantity_add.ts:32]: Add indent to lines from 32 to 35

File: src/app/actions/equipment/a_quantity_add.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          response as Equipment,
+          new HttpParams,
+          String(vs.objID)
+          )
+
```
Comment (Pablo203) [inline src/app/actions/equipment/a_quantity_add.ts:37]: Remove new line

File: src/app/actions/equipment/a_quantity_add.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    }),
+    tap(() => {
+      vs.refresh();
+    }),
+    defaultIfEmpty(null)
+    
+  );
+
+  
+  // switchMap(wizardDestroyed => apis.saveRecord(
```
Comment (Pablo203) [inline src/app/actions/equipment/a_quantity_add.ts:49]: Remove comment

File: src/app/actions/equipment/index.ts [hunk 5]
```diff
@@ -4,3 +4,4 @@ export * from './a_equipment_create';
 export * from './a_equipment_delete';
 export * from './a_equipment_reservation_create';
 export * from './a_equipment_reservation_delete';
+export * from './a_quantity_add';
```
Comment (Pablo203) [inline src/app/actions/equipment/index.ts:7]: Wrong file name. Should be :
```suggestion
export * from './a_equipment_quantity_add';
```

File: src/app/shared/config/actions.json.ts [hunk 6]
```diff
@@ -27,33 +31,23 @@ export const actionsConfig = [
...
         },
         //Edits
         "equipmentDetail": {
             actions: [
-
             ]
         },
         "equipmentTypeDetail": {
             actions: [
-
```
Comment (Pablo203) [inline src/app/shared/config/actions.json.ts:46]: Restore it to codebase

File: src/app/shared/config/actions.json.ts [hunk 7]
```diff
@@ -68,14 +62,6 @@ export const actionsConfig = [
         //Calendars
         "equipmentReservationsCalendar": {
             actions: [
-                {
```
Comment (Pablo203) [inline src/app/shared/config/actions.json.ts:71]: Restore it to codebase

File: src/app/shared/services/apiService/api.service.ts [hunk 8]
```diff
@@ -56,6 +56,8 @@ export class ApiService {
     params: HttpParams,
     objectID?: string
   ): Observable<HttpResponseType | null> {
+    console.log(objectID)
```
Comment (Pablo203) [inline src/app/shared/services/apiService/api.service.ts:59]: Remove redundant log

File: src/app/shared/services/apiService/api.service.ts [hunk 9]
```diff
@@ -56,6 +56,8 @@ export class ApiService {
...
+    console.log(body);
```
Comment (Pablo203) [inline src/app/shared/services/apiService/api.service.ts:60]: Remove redundant log

File: src/app/actions/equipment/a_quantity_add.ts [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+import { WizardService } from "src/app/shared/services/wizardService/wizard.service";
+import { switchMap, filter, tap, defaultIfEmpty, of } from "rxjs";
+import { CID } from "src/app/shared/config/constants.json";
+import { HttpParams } from "@angular/common/http";
+import { Equipment } from "src/app/shared/models/apiService.model";
+
+export function a_equipment_add_quantity(vs: ViewService, apis: ApiService, ws: WizardService) {
+  return of(vs.objID).pipe(
+    filter(objID => objID !== null),
+    switchMap(() => {
```
Comment (Pablo203) [inline src/app/actions/equipment/a_quantity_add.ts:12]: It'll look better as 
```suggestion
   switchMap(() => ws.create('addQuantity')),
```

File: src/app/actions/equipment/a_quantity_add.ts [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+      return ws.create('addQuantity')
+    }),
+   tap(() => {
```
Comment (Pablo203) [inline src/app/actions/equipment/a_quantity_add.ts:15]: Add one space more of indent

File: src/app/shared/config/views/wizard.json.ts [hunk 12]
```diff
@@ -1,5 +1,4 @@
-import { DROPDOWNS } from "../constants.json"
```
Comment (Pablo203) [inline src/app/shared/config/views/wizard.json.ts:1]: Restore it to codebase

File: src/app/shared/config/views/wizard.json.ts [hunk 13]
```diff
@@ -17,7 +16,7 @@ export const wizardConfig = [
                     fieldType: "text",
                     fieldName: "equipmentTypeId",
                     isClass: true,
-                    classDataUrl: DROPDOWNS.EQUIPMENT_TYPE,
```
Comment (Pablo203) [inline src/app/shared/config/views/wizard.json.ts:20]: Restore it to codebase

File: src/app/shared/config/views/wizard.json.ts [hunk 14]
```diff
@@ -60,56 +59,24 @@ export const wizardConfig = [
                 },
             ]
         },
-        "addEquipmentReservation": {
```
Comment (Pablo203) [inline src/app/shared/config/views/wizard.json.ts:63]: Restore it to codebase

File: src/app/shared/services/actionService/action.service.ts [hunk 15]
```diff
@@ -3,42 +3,36 @@ import { WizardService } from '../wizardService/wizard.service';
...
 import { ApiService } from '../apiService/api.service';
 import { ViewService } from '../viewService/view.service';
-
+import { ToastService } from '../toastService/toast.service';
 import {
   a_equipment_type_create,
   a_equipment_type_delete,
   a_equipment_create,
   a_equipment_delete,
-  a_equipment_reservation_create,
```
Comment (Pablo203) [inline src/app/shared/services/actionService/action.service.ts:12]: Restore it to codebase

File: src/app/shared/services/actionService/action.service.ts [hunk 16]
```diff
@@ -3,42 +3,36 @@ import { WizardService } from '../wizardService/wizard.service';
...
-
-  // EQUIPMENT
-  createEquipment() { return a_equipment_create(this.viewService, this.apiService, this.wizardService) }
-  deleteEquipment() { return a_equipment_delete(this.viewService, this.apiService, this.yesNoService) }
-
   // EQUIPMENT TYPE
   createEquipmentType() { return a_equipment_type_create(this.viewService, this.apiService, this.wizardService) }
   deleteEquipmentType() { return a_equipment_type_delete(this.viewService, this.apiService, this.yesNoService) }
 
-  // EQUIPMENT RESERVATION
```
Comment (Pablo203) [inline src/app/shared/services/actionService/action.service.ts:40]: Restore it to codebase

**Review by Pablo203 — CHANGES_REQUESTED** [review]

**Pull request merged**